nucypher/blockchain/eth/actors.py

```diff
@@ -1202,6 +1206,26 @@ def missing_commitments(self) -> int:
         missing = self.staking_agent.get_missing_commitments(checksum_address=staker_address)
         return missing
 
+    def verify_policy_payment(self, hrac: bytes) -> None:
+        arrangements = self.policy_agent.fetch_policy_arrangements(policy_id=hrac)
```

Comment:
Let's choose whether to call it HRAC or PolicyID

Reply:
What worries me is that we have `HRAC_LENGTH` (in `crypto.constants`) and `POLICY_ID_LENGTH` (in `blockchain.eth.constants`). Currently they're equal. What happens if we decide to change one of them? 

There would be a point to having both of them if we merely used HRAC as a policy ID, but also for other purposes (and even than, I'd prefer to have something like a `hrac_as_policy_id()` function), but now it *is* a policy ID, isn't it? We can just rename `TreasureMap.derive_hrac()` to `TreasureMap.derive_policy_id()` and not lose any meaning, can't we?

Reply:
Opened https://github.com/nucypher/nucypher/issues/2736